x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/connector_configuration.tsx

```diff
@@ -88,9 +88,9 @@ export const ConnectorConfiguration: React.FC = () => {
     ({ serviceType }) => serviceType === connector.service_type
   )?.docsUrl;
 
-  const isBeta =
-    !connector.service_type ||
```

Comment:
`customized_connector` by default has `null` service type, so it was a trick to mark this as beta, we can get rid of it now